src/js/common/helpers/login-helpers.js

```diff
@@ -29,7 +29,9 @@ export function getVerifyUrl(onUpdateVerifyUrl) {
   }).then(response => {
     return response.json();
   }).then(json => {
-    onUpdateVerifyUrl(json.identity_proof_url);
+    if (json.identity_proof_url) {
```

Comment:
It's more "correct" probably to check response.ok and throw an error if not, but that would leave uncaught promise rejections. We don't appear to need to handle rejections, so this seemed like a reasonable way to avoid the errors.